erfa/tests/test_erfa.py

```diff
@@ -19,6 +19,28 @@ def __init__(self, *args, **kwargs):
             pass
 
 
+def embedded_liberfa(path=erfa.ufunc.__file__):
+    import platform
+    import subprocess
+
+    lddcmd = ['ldd']
```

Comment:
When I tried to check somewhat similarly on travis, I used `nm` to check the status perhaps more directly:
https://github.com/liberfa/pyerfa/blob/822d095c96e8203d222be23450beb83dc792b876/.travis.yml#L97 (here `-u` only lists undefined objects, which should be the case if the system library was used; one might want `--defined-only` for the reverse). Maybe this has an equivalent on windows too?

Reply:
nice, the command is the same both on GNU/Linux and Mac OS X, I will update.